Review for Portuguese translation of `maintenance_contributions.pt.Rmd` (#721)
ropensci/dev_guide: maintenance_contributing.pt.Rmd

---
repo-actions: true
aliases: 
  - contributingguide.html
---

# Guia de contribuição {#contributingguide}

```{block, type="summaryblock"}
Este capítulo apresenta o nosso Guia de Contribuição, que descreve como você pode fazer contribuições com código e sem código para a rOpenSci.

```
-

> Então você quer contribuir para a rOpenSci? Fantástico! Nós desenvolvemos o [Guia de contribuição da comunidade rOpenSci](https://contributing.ropensci.org/) para dar as boas-vindas a você na rOpenSci e ajudá-lo a se reconhecer como um colaborador em potencial. O guia ajudará você a descobrir o que você pode ganhar doando seu tempo, conhecimento e experiência, combinando suas necessidades com coisas que ajudarão a missão da rOpenSci, e conectando você a recursos que podem te ajudar ao longo do caminho.

Reviewer: ```suggestion
Então você quer contribuir para a rOpenSci? Fantástico! Nós desenvolvemos o [Guia de contribuição da comunidade rOpenSci](https://contributing.ropensci.org/) para dar as boas-vindas a você na rOpenSci e te ajudar a se reconhecer como um(a) colaborador(a) em potencial. O guia ajudará você a descobrir o que você pode ganhar doando seu tempo, conhecimento e experiência, combinando suas necessidades com coisas que ajudarão a missão da rOpenSci, e conectando você a recursos que podem te ajudar ao longo do caminho.
```

Author: Good suggestion.

> Nossa equipe e comunidade promovem ativamente um ambiente acolhedor em que usuários e desenvolvedores de diferentes origens e níveis de habilidade aprendem, compartilham ideias e inovam juntos abertamente por meio de normas e software compartilhados. A participação em todas as atividades da rOpenSci é apoiada por [nosso Código de Conduta](https://ropensci.org/code-of-conduct).

Reviewer: ```suggestion
Nossa equipe e comunidade promovem ativamente um ambiente acolhedor em que pessoas usuárias e desenvolvedoras de diferentes origens e níveis de habilidade aprendem, compartilham ideias e inovam juntas abertamente por meio de normas e software compartilhados. A participação em todas as atividades da rOpenSci é apoiada por [nosso Código de Conduta](https://ropensci.org/code-of-conduct).
```

> Aceitamos contribuições com código e sem código de programadores novos e experientes em qualquer estágio da carreira e em qualquer setor. Você não precisa ser um desenvolvedor! Talvez você queira **passar 30 minutos** compartilhando o caso de uso do seu pacote em nosso fórum público ou relatando um bug, **uma hora** você pode aprender participando de uma chamada da comunidade, **cinco horas** revisando um pacote de R enviado para revisão aberta por pares, **ou talvez você queira assumir um compromisso contínuo** para ajudar a manter um pacote.

Reviewer: ```suggestion
Aceitamos contribuições com código e sem código de pessoas programadoras novas ou experientes em qualquer estágio da carreira e em qualquer setor. Você não precisa ser um(a) desenvolvedor(a)! Talvez você queira **passar 30 minutos** compartilhando o caso de uso do seu pacote em nosso fórum público ou relatando um bug, **uma hora** aprendendo e participando de uma chamada da comunidade, **cinco horas** revisando um pacote de R enviado para revisão aberta por pares, **ou talvez você queira assumir um compromisso contínuo** para ajudar a manter um pacote.
```

- Faça e responda a novas perguntas de pesquisa conhecendo novas ferramentas de software e aliados

> - Sentir-se confiante e apoiado em seus esforços para escrever código e desenvolver software

Reviewer: ```suggestion
- Sentir-se confiante e com apoio em seus esforços para escrever código e desenvolver software
```
